.github/workflows/ci-minimized.yml

```
@@ -0,0 +1,254 @@
+        env:
+            PROGRESS_TYPE: plain
+            SPRYKER_PLATFORM_IMAGE: spryker/php:8.3
+            SPRYKER_CURRENT_REGION: EU
+            SPRYKER_DYNAMIC_STORE_MODE: true
+            TRAVIS: 1
+
+        steps:
+            -   uses: actions/checkout@v4
+
+            -   name: Install docker-compose
+                run: |
+                    sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+                    sudo chmod +x /usr/local/bin/docker-compose
+
+            -   name: Docker bootstrap
+                run: |
+                    git clone https://github.com/spryker/docker-sdk.git ./docker
+                    docker/sdk boot deploy.ci.fe-assets-api.mariadb.yml
+
+            -   name: Run docker up testing
+                run: |
+                    docker/sdk up -t
+
+            -   name: Run docker build development
+                run: |
+                    docker/sdk cli vendor/bin/console transfer:databuilder:generate
+                    docker/sdk cli vendor/bin/console rest-api:generate:documentation
+
+            -   name: Composer dump-autoload
+                run: |
+                    docker/sdk cli composer dump-autoload -o -a --apcu
+
+            -   name: Demo-data
+                run: |
+                    docker/sdk cli vendor/bin/console queue:setup
+                    docker/sdk cli vendor/bin/console setup:init-db
+                    docker/sdk cli vendor/bin/console data:import --config=data/import/common/minimal.yml
+                    docker/sdk cli vendor/bin/console search:setup:sources
+                    docker/sdk cli vendor/bin/glue glue-api:controller:cache:warm-up
+                    docker/sdk cli vendor/bin/glue api:generate:documentation
+
+            -   name: Publish & Sync
+                run: |
+                    docker/sdk testing console queue:worker:start --stop-when-empty
+
+            -   name: Fixtures
+                run: |
+                    docker/sdk testing codecept fixtures
+
+            -   name: Glue Acceptance CheckoutMinimumWorkflow
+                run: |
+                    docker/sdk testing codecept run -c codeception.api.yml -g CheckoutMinimumWorkflow
+
+            -   name: Yves Acceptance CheckoutMinimumWorkflow
+                run: |
+                    docker/sdk testing codecept run -c codeception.acceptance.yml -g YvesCheckoutMinimumWorkflow
+
+            -   name: Save PHP logs
+                if: failure()
+                run: |
+                    mkdir -p logs
+                    docker logs spryker_ci_backoffice_eu_1 > logs/backoffice_app.log 2>&1
+                    docker logs spryker_ci_glue_eu_1 > logs/glue_app.log 2>&1
+                    docker logs spryker_ci_yves_eu_1 > logs/yves_app.log 2>&1
+                    docker logs spryker_ci_merchant_portal_eu_1 > logs/merchant_portal_app.log 2>&1
```

Only for executed tests
```suggestion
                    docker logs spryker_ci_glue_eu_1 > logs/glue_app.log 2>&1
                    docker logs spryker_ci_yves_eu_1 > logs/yves_app.log 2>&1
```